synfig-studio/plugins/lottie-exporter/settings.py

```diff
@@ -68,10 +71,11 @@
 PRE_COMP_LAYER = {"rotate", "zoom", "translate", "stretch"}
 GROUP_LAYER = {"group", "switch"}
 SKELETON_LAYER = {"skeleton"}
+TEXT_LAYER = {"text"}
 UNKNOWN_LAYER = "unknown_layer"
 CONVERT_METHODS = {"add", "atan2","average", "bone", "bone_link", "bone_root", "composite", "cos", "exp", "fromint", "linear", "radial_composite", "scale", "sine", "subtract", "switch", "vectorangle","weighted_average"}
 BONES = {"bone", "bone_root"}
-
+FONT_STYLES = ["Sans Serif","Times New Roman","Calibria","Arial","Courier","Comic Sans"]
```

Comment:
What about other font styles? Are they supported?

Reply:
I checked the fonts in the list available in #1432 and the overlap between these fonts and lottie format is only these fonts. More fonts are supported by Lottie but I did not find them in the Synfig font list.